### PR TITLE
More body markings!

### PR DESCRIPTION
**Pull request by Kegdo-Jr**
## About The Pull Request
Adds the following markings, available for any species (except digitigrade havers, I couldn't be bothered):

`Chestplate, hip, elbow, shoulder, glove, sleeve, sock, longsock, vertical stripe, fangs, lips, facemask, facedisc`

Mostly aimed at humans/FBPs, but any species can use them.

Will conflict with #1249 . Considering that this PR adds ~~more markings, it's more important~~ a new body_marking file for `modular_zubbers`, with a dedicated `icon`, it should be merged first.

изображение

## Why It's Good For The Game

изображение

изображение

Not enough markings.

## Proof Of Testing

Every single added marking, each a different color.

65d79c94a757fac1

2a389ebe2329d738

e24ff07726a0fc65

Markings being put to good use (demipanda, woahhh)

изображение

## Changelog

:cl: Kegdo
add: more markings: Chestplate, hip, elbow, shoulder, glove, sleeve, sock, longsock, vertical stripe, fangs, lips, facemask, facedisc.
/:cl:

### CODE REVIEW

**Comment by iskawhiskers** [comment]
Drafting #1249 to rebase off this one

File: modular_zubbers/code/modules/mob/dead/new_player/body_markings.dm [hunk 1]
```diff
@@ -0,0 +1,82 @@
+/*
+	Bubberstation body markings
+	Body marking code is in modular_skyrat/[...]/new_player/body_markings.dm
+	Template below
+
+/datum/body_marking/bubber/111 //TEST ME
+	name = "111"
+	icon_state = "111"
+	affected_bodyparts = HEAD | CHEST | ARM_LEFT | ARM_RIGHT | HAND_LEFT | HAND_RIGHT | LEG_LEFT | LEG_RIGHT | FOOT_LEFT | FOOT_RIGHT
```
Comment (Majkl-J) [inline modular_zubbers/code/modules/mob/dead/new_player/body_markings.dm:9]:
```suggestion
```

**Review by Majkl-J — CHANGES_REQUESTED** [review]
Amazing work, bonus points if you find any other bubber markings and refactor them into this format. You don't have to though, plus I'm not even sure if there are any.

Just commit the one suggestion and green checkmark will be given

**Comment by Majkl-J** [comment]
> image

I don't enjoy busting down the door smacking my ass right in the middle of someone's code, but fair

**Review by Majkl-J — APPROVED** [review]

**Pull request merged**